Multisig/Data/Persistence/CoreDataStack.swift

```diff
@@ -36,6 +39,18 @@ class CoreDataStack {
                  Check the error message to determine what the actual problem was.
                  */
                 fatalError("Unresolved error \(error), \(error.userInfo)")
+            } else {
+                // merge changes from background contexts into the view context
+                NotificationCenter.default
+                .publisher(for: Notification.Name.NSManagedObjectContextDidSave)
+                .receive(on: RunLoop.main)
+                .sink { [unowned container] notification in
+                        let savedMOC = notification.object as! NSManagedObjectContext
+                        guard savedMOC != container.viewContext,
+                            savedMOC.persistentStoreCoordinator == container.persistentStoreCoordinator else { return }
+                        container.viewContext.mergeChanges(fromContextDidSave: notification)
```

Comment:
Merging into viewContext on _every_ save might cause UI performance issues in the future when we do background synchronization. Let's see.

Reply:
this will update only the loaded models in the context, as far as I understood. I'm a newbie in Core Data, don't know what will happen in the future :)